### PR TITLE
Refo 16: Job Save logic

### PR DESCRIPTION
**Pull request by makersmelx**

### CODE REVIEW

File: src/components/job/JobList.jsx [hunk 1]
```diff
@@ -26,38 +26,35 @@ export default function JobList() {
...
-                    <div className="flex felx-row w-11/12">
-                      <p className="w-1/2 truncate">
-                        {job.benefits.hasMedical
-                          ? "Medical Benefits"
-                          : "No Medical Benefits"}
-                      </p>
-                      <p className="w-1/2 truncate">
-                        Minimum Pay: {job.wage.min} {job.wage.type}
-                      </p>
+                    <div className="flex flex-row flex-none">
```
Comment (makersmelx) [inline src/components/job/JobList.jsx:40]: @kendebacker some changes here. could you take a look at the demo page to see the changes in display?

File: src/components/job/JobSave.jsx [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+        return <Spin className="h-8 w-8" />;
+      }
+      if (checkIsJobSaved(jobId)) {
+        return <StarIcon fontSize="inherit" onClick={onClick} />;
+      } else {
+        return <StarOutlineIcon fontSize="inherit" onClick={onClick} />;
+      }
+    }, [saveJob.loading, checking, jobsSaved]);
+
+    return <div className="text-4xl">{starIcon}</div>;
```
Comment (makersmelx) [inline src/components/job/JobSave.jsx:40]: @kendebacker refer here for how to change the icon size using `text-size`.  `fontSize="inherit"` and then wrap with a `<div>`

File: src/components/job/JobList.jsx [hunk 3]
```diff
@@ -26,40 +26,35 @@ export default function JobList() {
...
+                          {job.benefits.hasMedical
+                            ? "Medical Benefits"
+                            : "No Medical Benefits"}
+                        </p>
+                        <p className="truncate w-1/2">
+                          Minimum Pay: {job.wage.min} {job.wage.type}
+                        </p>
+                      </div>
                     </div>
                   </div>
```
Comment (makersmelx) [inline src/components/job/JobList.jsx:52]: @kendebacker some changes here. could you take a look at the demo page to see the changes in display?

File: src/components/job/JobSave.jsx (hunk 2 above)
Comment (kendebacker) [reply src/components/job/JobSave.jsx:40]: Ah nice, I will update my existing icons in a new PR

File: src/components/job/JobList.jsx (hunk 3 above)
Comment (kendebacker) [reply src/components/job/JobList.jsx:52]: Ah yes the flex-none is nice to keep the listing size constant!

**Review by kendebacker — APPROVED** [review]

**Pull request merged**